draft-ietf-tls-tls13.md

```diff
@@ -1890,14 +1895,16 @@ Cookies serve two primary purposes:
 When sending a HelloRetryRequest, the server MAY provide a "cookie" extension to the
 client (this is an exception to the usual rule that the only extensions that
 may be sent are those that appear in the ClientHello). When sending the
-new ClientHello, the client MUST echo the value of the extension.
-Clients MUST NOT use cookies in subsequent connections.
+new ClientHello, the client MUST echo the value of the extension and should
+subsequently delete the local copy of the cookie value.
+Clients MUST NOT reuse cookie values in subsequent connections.
 
 
 ###  Signature Algorithms
 
 The client uses the "signature_algorithms" extension to indicate to the server
-which signature algorithms may be used in digital signatures. Clients which
+which signature algorithms may be used in digital signatures over TLS protocol
```

Comment:
okay.
(I originally noted this spot on my printout because I had it in my head that it only applied to the TLS protocol structures, but the following text seemed to make it apply to certificates as well.)